engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -345,26 +404,44 @@ public Vector3f centerPointForNormal(Vector3f normal) {
         return new Vector3f();
     }
 
+    /**
```

Comment:
Description. Same for all other min functions

Reply:
We should keep either the description or the return statement (my favorite would be the return statement and an empty method description).
Redundancy is always a bad thing ;) keep it DRY.

Same for all further min.. and max.. methods.

Reply:
Definitely have to agree. Will keep that in mind.